dump/plugins/port.py

```diff
@@ -0,0 +1,81 @@
+from .executor import Executor
+from dump.match_infra import MatchEngine, MatchRequest
+from dump.helper import create_template_dict
+
+class Port(Executor):
+    
+    ARG_NAME = "port_name"
+    
+    def __init__(self):
+        self.match_engine = MatchEngine()
+        self.ret_temp = {}
+        self.ns = ''
+          
+    def get_all_args(self, ns=""):
+        req = MatchRequest(db="CONFIG_DB", table="PORT", key_pattern="*", ns=ns)
+        ret = self.match_engine.fetch(req)
+        all_ports = ret["keys"]
+        return [key.split("|")[-1] for key in all_ports]
+            
+    def execute(self, params_dict):
```

Comment:
@vivekreddynv why do we have this: `params` -> `params_dict`?

Reply:
Addressed